frontend/src/Onboarding/LandingScreen.tsx

```diff
@@ -196,28 +164,10 @@ export class Onboarding extends React.Component<Props> {
         <Header>
           <h1>GraduateNU</h1>
           <LoginButtonContainer>
-            <a
-              href="https://admin.khoury.northeastern.edu"
-              style={{ textDecoration: "none" }}
-            >
-              <ColorButton variant="contained">Get Started</ColorButton>
-            </a>
-            {this.dev && (
-              <>
-                <ColorButton
-                  variant="contained"
-                  onClick={this.onDevStudentClick.bind(this)}
-                >
-                  Dev Bypass (Student)
-                </ColorButton>
-                <ColorButton
-                  variant="contained"
-                  onClick={this.onDevAdvisorClick.bind(this)}
-                >
-                  Dev Bypass (Advisor)
-                </ColorButton>
-              </>
-            )}
+            <PrimaryLinkButton to="/login" style={{ marginRight: "1em" }}>
+              Login
+            </PrimaryLinkButton>
+            <PrimaryLinkButton to="/onboarding">Sign Up</PrimaryLinkButton>
```

Comment:
these two buttons should be the same size even though the text is different in length. I think you can set a min width in the css